crypten/mpc/mpc.py

```diff
@@ -218,6 +231,111 @@ def to(self, *args, **kwargs):
             self.share = share
             return self
 
+    def add(self, y):
+        return self.clone().add_(y)
+
+    def add_(self, y):
+        """added addition according to MAC"""
+
+        if self._mac is not None:
+            private = isinstance(y, MPCTensor)
+
+            if not private:
+                self._tensor += y
+                scaled_y = self._tensor.encoder.encode(y)
+                self._mac += self.alpha * scaled_y
+            else:
+                self._tensor += y._tensor
+                self._mac += y._mac
```

Comment:
Still need to remove most of this block from the `if self._mac is not None`.

(At least lines 244 and 248).